target/scripts/start-mailserver.sh

```diff
@@ -170,24 +169,35 @@ function _register_functions() {
 # ? >> Executing all stacks / actual start of DMS
 # ------------------------------------------------------------
 
-_early_supervisor_setup
-_early_variables_setup
+# Ensure DMS only adjusts config files for a new container.
+# Container restarts should skip as they retain the modified config.
+if [[ ! -f /CONTAINER_START ]]; then
```

Comment:
I think reading the non-negated statement is easier. Because we have an `else`, having `-f /CONTAINER_START` is feasible.

Reply:
I did it that way, because I wanted the "fresh container" routines to come first and the "container was restarted" stuff later in the else block.
We can change that, but the reasoning that the if statement is hard to read because of the negation is thin 😉 (IMO).

Reply:
Absolutely fine with me :)